asset_stream_manager/cdc_fuse_manager.cc

```diff
@@ -29,7 +29,7 @@ namespace {
 constexpr char kFuseFilename[] = "cdc_fuse_fs";
 constexpr char kLibFuseFilename[] = "libfuse.so";
 constexpr char kFuseStdoutPrefix[] = "cdc_fuse_fs_stdout";
-constexpr char kRemoteToolsBinDir[] = "/opt/developer/tools/bin/";
+constexpr char kRemoteToolsBinDir[] = "~/.cache/cdc_file_transfer/";
```

Comment:
Note that `~` is expanded by the shell running on the instance. That should work when using SCP, but there can be situations in which `$HOME` works but `~` does not. If it works right now, then keep it.

Reply:
Yeah, I think for that reason I had to remove quoting from remote_util.cc. IIRC I used wordexp at some point in the server, but since I don't see that anymore, I guess I have removed it again. Anyway, this works for both Stadia and gLinux.

Reply:
That being said, it would make sense to run through some corner cases later and/or adapt the e2e tests to make sure everything still works.